### PR TITLE
show Lantern version on outdated page

### PR DESCRIPTION
**Pull request by joesis**
It's for https://github.com/getlantern/lantern-internal/issues/691#issuecomment-288780515, depends on https://github.com/getlantern/lantern-build/issues/202 which put the required `version-beta.txt` to S3.

### CODE REVIEW

File: getlantern.org/static/js/main.js [hunk 1]
```diff
@@ -1,57 +1,20 @@
 $(document).ready(function(){
 
-  var ANDROID_LINK = "lantern-installer-beta.apk";
```
Comment (joesis) [inline getlantern.org/static/js/main.js]: This, and below download related functions, are moved to `download.js`, to be reused in both `index.html` and `outdated.html`

File: getlantern.org/static/js/download.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+$(document).ready(function(){
+
+  var ANDROID_LINK = "lantern-installer-beta.apk";
+
+  var set_download_link = function() {
+    var os = platform.os.architecture + platform.os.family;
+    var os_links = [
+      {regexp: "[2-6]{2}[Ww]indows", link: "lantern-installer-beta.exe"},
+      {regexp: "[2-6]{2}OS X", link: "lantern-installer-beta.dmg"},
+      // for Android devices with large screen
```
Comment (joesis) [inline getlantern.org/static/js/download.js]: Note that the download link for Android is move above Linux, as the os can be `32Android` sometimes.

File: getlantern.org/static/js/download.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    $("#download-android-button").attr("href", ANDROID_LINK);
+    if (platform.os.family === "Android") {
+      var downloadBtn = document.getElementById("download-button");
+      $("#download-android").css("display", "block");
+      $("#other-systems").css("display", "block");
+      $("#download-android-button").after(downloadBtn);
+    }
+  };
+
+  var update_version_number = function() {
```
Comment (joesis) [inline getlantern.org/static/js/download.js]: This fetches the version number and shows it. It doesn't affect `index.html` as relevant elements are absent there.

**Comment by joesis** [comment]
@quension Could you have a look?

I pushed the code to beta but the site has an expired cert.

The `build` folder is annoying when reviewing code. Can we simply remove it from the repo consider `Makefile` always rebuild?

**Comment by atavism** [comment]
Deployed here: https://getlantern.org/outdated.html

I manually uploaded a `version-beta.txt` to the `getlantern.org` bucket until we do new a release.

I also continued testing with the staging proxy; after encountering the `outdated` page, I was able to download/install the new APK without issue

**Comment by joesis** [comment]
Great thanks @atavism !

**Pull request merged**